src/main/java/frc/robot/subsystems/AgitatorSubsystem.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.config.Config;
+import frc.robot.config.FluidConstant;
+
+public class Agitator extends SubsystemBase {
+
+    private VictorSPX agitatorMotor;
+
+
+public Agitator (VictorSPX agitatorMotor)
```

Comment:
Couple of things here - one, all other subsystems were defined using a singleton pattern - can you have a look at one of the existing ones and follow that pattern?

Also, your constructor assumes someone else will instantiate the VictorSPX motor controller object and I think the convention we've used elsewhere is for the subsystems to each instantiate and manage their own. You'll also need to know the CAN ID of the Victor, which you'll pull from the robot constants.

Reply:
will do, ill get it done by wednesday

Reply:
Here is a reference to the preferred Singleton pattern 
https://github.com/georgetzavelas/guava-examples/blob/master/src/com/design/pattern/creational/singleton/Singleton.java